packages/components/button/stories/button.stories.ts

```diff
@@ -0,0 +1,30 @@
+import type { Meta, StoryFn } from "@storybook/vue3";
+import { LButton } from "../src";
+
+export default {
+  title: "Components / Button",
+  component: { LButton },
+} as Meta<typeof LButton>;
+
+const Template: StoryFn<typeof LButton> = (args: any) => ({
+  components: { LButton },
+  setup() {
+    return { args };
+  },
+  template: `
+  <LButton colorScheme='teal' size='xs'>
+    Button
+  </LButton>
+  <LButton colorScheme='teal' size='sm'>
+    Button
+  </LButton>
+  <LButton colorScheme='teal' size='md'>
+    Button
+  </LButton>
+  <LButton colorScheme='teal' size='lg'>
+    Button
+  </LButton>
+  `,
+});
+
```

Comment:
with another case? Like the left icon and right icon,.... we need to test the full case same docs of chakra with the button. This thing will help us avoid many issues

Reply:
As with other components, mate! @mahuudu